x/concentrated-liquidity/keeper_test.go

```diff
@@ -24,50 +26,61 @@ import (
 )
 
 var (
-	DefaultMinTick, DefaultMaxTick                       = types.MinTick, types.MaxTick
```

Comment:
Note to reviewer: this diff block is mostly due to reformatting/linting after a comment was added. Only `DefaultLiquidityAmt` and `FullRangeLiquidityAmt` were actually changed.